Core/Saves/ATBTimer.cs

```diff
@@ -66,26 +66,27 @@ public void FirstTurn()
 
         public bool Update()
         {
-            if (Damageable != null)
+            if (Damageable == null || Damageable.IsDead)
             {
-                if (!Damageable.IsDead)
```

Comment:
i just noticed the isdead is in the wrong place. I fixed it on my branch. Should be `if (_damageable.IsDead && ATBBarPos > 0)` having the ATBBarPos if by itself always sets value to 0.